packages/next/pages/_error.tsx

```diff
@@ -57,11 +57,8 @@ export default class Error<P = {}> extends React.Component<P & ErrorProps> {
                 title
               ) : (
                 <>
-                  Application error: a client-side exception has occurred (
-                  <a href="https://nextjs.org/docs/messages/client-side-exception-occurred">
-                    developer guidance
-                  </a>
-                  )
+                  Application error: a client-side exception has occurred (see
```

Comment:
Is this browser dev console or the terminal console running next cli?

Reply:
Updated to stress it's the browser console